.github/workflows/release-docker.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Docker Image
+on: 
+  release:
+    types: 
+      - published
+
+jobs:
+
+  release:
+    name: Docker build and push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: master
```

Comment:
Is it needed?

Reply:
yes, the workflow is triggered not on push but on release event, so it doesn't know from the context what branch to checkout unfortunately 😞

Reply:
Ah yes 🤦🏼  Forgot about that, thanks!